packages/core/resolve-command/src/index.ts

```diff
@@ -426,10 +532,10 @@ const executeCommand = async (pool, { jwtToken: jwt, ...command }) => {
       aggregateVersion: aggregateVersion + 1,
       timestamp: Math.max(minimalTimestamp + 1, Date.now()),
       type: event.type,
-      payload: event.payload
+      payload: event.payload || {}
```

Comment:
Breaking change 1: formerly valid JSON payloads aka `false`, `0`, `null` and `""` coerced to object